dmoj/executors/ZIG.py

```diff
@@ -0,0 +1,27 @@
+from dmoj.executors.compiled_executor import CompiledExecutor
+
+
+class Executor(CompiledExecutor):
+    ext = 'zig'
+    name = 'ZIG'
+    command = 'zig'
+    test_program = '''
+const std = @import("std");
+
+pub fn main() !void {
+    const stdout = std.io.getStdOut().outStream();
+    try stdout.print("echo: Hello, {}!\\n", .{"World"});
```

Comment:
If Zig has a way to read all lines of stdin, it should be used here rather than hardcoding the output of the self test.

Reply:
added, though this does depend on trailing new lines I believe